citylearn/electric_vehicle_charger.py

```diff
@@ -0,0 +1,374 @@
+import inspect
+from typing import List, Dict
+from citylearn.base import Environment
+
+from citylearn.electric_vehicle import electric_vehicle
```

Comment:
class names should be written in uppercase first letter for each word and underscores; see other class names.